README.md

```diff
@@ -1,2 +1,12 @@
 # stardog-userrole-operator
 A Kubernetes operator to manage Stardog users and roles.
+
+# Generating the REST client
+
+The package stardogrest is a REST client generated by [autorest](http://azure.github.io/autorest/) based on the [stardogrest/stardog_swagger.yaml](stardogrest/stardog_swagger.yaml) file. If the stardog REST API changes, the [stardogrest/stardog_swagger.yaml](stardogrest/stardog_swagger.yaml) should be updated to reflect the changes, and then autorest should be run again with the following command:
+
+```
+autorest --go --package-name=stardogrest --input-file=stardogrest/stardog_swagger.yaml --output-folder=stardogrest
```

Comment:
What happens if you execute like this?
```suggestion
autorest --go --package-name=github.com/vshn/stardogrest --input-file=stardogrest/stardog_swagger.yaml --output-folder=stardogrest
```

Reply:
Actually, yes, that fixes the import problem. And I dug around in the command-line options and found one that adds the package name correctly in the generated code. So I can probably put it in the Makefile after all.

Do you think it should go in the Makefile or just keep it as a manual process in the README?

Reply:
BTW, the generated code fails the linting step, so maybe it's better to leave it as a manual step...

Reply:
What linting exactly? `go fmt` ? There's nothing wrong with executing fmt after the generation

Reply:
Regarding linting: It may be worth checking out https://golangci-lint.run/, where it's possible to combine all sorts of linters and also exclude directories

Reply:
Yeah, that's a good point. As I wrote in the README, it's a simple question of running `make lint` after generating the code, and it automatically formats it correctly. So I guess I should move this step to the Makefile..?

Reply:
ah, so something like works as well? You could make the `generate` target depend on `autorest` and since `build` depends on `generate` as well as `fmt / vet`, it would generate and then format it
```
autorest:
  autorest ... (maybe needs go-get autorest first...?)
...
generate: controller-gen autorest
  ...
```
you could also add a go vet/fmt commands right in the `autorest` target too, after executing the autorest command, but I think it's fine if the formatting/vetting is a separate step.

Reply:
That's a very good idea. I am currently adding it to the Makefile. Autorest has to be installed via npm, but it appears that it's possible to do it cleanly in a makefile. I am following this blog post about how to do it: https://blog.jayway.com/2013/05/19/auto-install-node-js-and-npm-from-makefile/

Reply:
I have added the autorest code generation to the Makefile as you suggested. Can you please have a look and see if you recommend any changes to this implementation.

If you agree, I will go ahead and remove the manual instructions from the README and fix the go.sum merge conflict.

Reply:
The autorest code generation is now in the Makefile and the README is updated accordingly.